dbstore.go

```diff
@@ -482,6 +504,11 @@ func (p *persister) update(ws *Worksheet) error {
 				orphanedChildren[index] = append(orphanedChildren[index], childWs.Id())
 			}
 		}
+		if _, ok := change.after.(*Slice); !ok {
```

Comment:
I think this'd be better as an if/else with the `if sliceAfter, ok := change.after.(*Slice)...` block below...it took me a minute to notice the `!` in `!ok`

Reply:
Here, because we need to handle both orphaned & adopted children in the non-slice case, I simply added comments `// slice value` and `// non-slice value`